packages/paste-core/components/skeleton-loader/src/index.tsx

```diff
@@ -59,6 +62,7 @@ const SkeletonLoader = React.forwardRef<HTMLDivElement, SkeletonLoaderProps>(
         borderTopLeftRadius={borderTopLeftRadius}
         borderTopRightRadius={borderTopRightRadius}
         display={display}
+        element={element}
```

Comment:
Good question. I think mainly the reason is going to be that because it just extends a bunch of Box props, validating those is really expensive.

We used to do it on Box itself, and it turned out to be such a performance hit in development for engineers we removed them. Having to look up the values of these props on the theme is pretty expensive performance wise.

Other reasoning is that the values are part of the design tokens theme, so there is a reference for you to get the right thing